package.json

```diff
@@ -2,35 +2,43 @@
   "name": "oniyi-vcard-parser",
   "version": "1.0.0",
   "description": "A vcard to JSON and back parser",
+  "license": "MIT",
   "author": {
     "name": "Benjamin Kroeger",
     "email": "benjamin.kroeger@gmail.com"
   },
-  "repository": "benkroeger/oniyi-vcard-parser",
-  "license": "MIT",
-  "files": [
-    "index.js"
-  ],
   "keywords": [
     "oniyi-vcard-parser",
     "oniyi",
     "vcard",
     "parser"
   ],
+  "repository": "benkroeger/oniyi-vcard-parser",
+  "scripts": {
+    "format": "prettier-eslint --write \"lib/**/*.js\" \"meta/**/*.js\" \"test/**/*.js\"",
```

Comment:
meta doesn't exist here. assuming copy&paste error from one of the gravity components?!

Reply:
you assumed well. I'll remove meta